### PR TITLE
Add ktlint to our setup

### PR DESCRIPTION
**Pull request by Pike**

### CODE REVIEW

**Comment by Pike** [comment]
CC @MikkCZ

File: fluent.syntax/.editorconfig [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[src/test/**/*.{kt,kts}]
+# Allow wildcard imports in tests.
+disabled_rules=no-wildcard-imports
+
+[src/main/kotlin/org/projectfluent/syntax/serializer/Error.kt]
```
Comment (MikkCZ) [inline fluent.syntax/.editorconfig:5]: I guess it doesn't like that the name of the file does not match the name of the single class inside, right?

Comment (Pike) [reply fluent.syntax/.editorconfig:5]: Exactly.

File: fluent.syntax/src/main/kotlin/org/projectfluent/syntax/parser/Parser.kt [hunk 2]
```diff
@@ -1,6 +1,8 @@
 package org.projectfluent.syntax.parser
 
+/* ktlint-disable no-wildcard-imports */
```
Comment (MikkCZ) [inline fluent.syntax/src/main/kotlin/org/projectfluent/syntax/parser/Parser.kt:3]: My personal preference would be to use single line comment.
```kotlin
import org.projectfluent.syntax.ast.* // ktlint-disable no-wildcard-imports
```

File: fluent.syntax/.editorconfig (hunk 1 above)
Comment (MikkCZ) [reply fluent.syntax/.editorconfig:5]: Interesting, I wouldn't consider it as an issue in Kotlin itself, however obeying that rule may help, especially for Java developers, to orient in the code quickly. The proper name for the class may be plural `Errors` if you decide to keep all the related exceptions there in the future.

**Pull request merged**